.travis.yml

```diff
@@ -4,31 +4,32 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
 env:
-  - SYMFONY_VERSION="2.7.*,>2.7.6"
+  - SYMFONY_VERSION="3.0.*"
```

Comment:
right .. guess I will then force 2.8 for now and then list 3.0 in place of 2.8 below